apps/sandbox/package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "@expo/sandbox",
   "version": "0.0.0",
-  "main": "expo-router/entry",
   "scripts": {
     "start": "expo start",
-    "android": "expo start --android",
-    "ios": "expo start --ios"
```

Comment:
good to keep expo-router entry as well as running from expo-go by default.
also add other expo-router deps like safe-area-context if you want to test it using dev builds.